scripts/laz_align.py

```diff
@@ -119,19 +119,20 @@ def laz_align(work_dir,
                 {ref_dem} {input_laz}   \
                 -o {transform_pc}', log)
 
-    # Grid the output to a 1 meter tif
+    # Grid the output to a 0.5 meter tif (NOTE: this needs to be changed to 1m if using py3dep)
     point2dem_func = join(asp_dir, 'point2dem')
     final_tif = join(work_dir, 'pc-grid', 'run')
     cl_call(f'{point2dem_func} {transform_pc}-trans_source.laz \
-                --dem-spacing 1 --search-radius-factor 2 -o {final_tif}', log)
-    
-    if not exists(final_tif):
-        log.info('No final product created')
-        return 1
+                --dem-spacing 0.5 --search-radius-factor 2 -o {final_tif}', log)
+
+    # For some reason this is returning 1 when a product IS created..
+    #if not exists(final_tif):
+    #    log.info('No final product created')
+    #    return 1
 
     return final_tif
 
 
 # To run For now just run this script
 if __name__ == '__main__':
```

Comment:
This just lets us run the script on its own. Otherwise call python laz_align.py won't do anything (most for testing)